apps/website/src/components/Playground/code/app.js

```diff
@@ -1,11 +1,18 @@
 import hljs from 'highlight.js/lib/core';
-import 'highlight.js/styles/vs.css';
+import 'highlight.js/styles/vs2015.css';
```

Comment:
Also changing a theme for preview in try out.

Reply:
nit: Can you add this to the description?